### ISSUE TITLE
scheduler.step() has inconsistent behaviour with pytorch versions. Getting the right LR when resuming an experiment is annoying and needs work.

### ISSUE THREAD

**Issue by learning-luke** [BayesWatch/pytorch-experiments-template#6]
This gave me a huge headache for a time. On some version of pytorch if you pass the epoch=epoch argument in it fast forwards the scheduler to the last value, and does so differently depending on which type of scheduler.
